packages/Algernon.package/ALGBreadcrumbs.class/instance/initializeDefaultCrumb.st

```diff
@@ -1,8 +1,8 @@
 initialization
 initializeDefaultCrumb
 
-	| crumb |
-	crumb := self createCrumb: ALGCore title.
-	crumb center: self center.
-	crumb position: (self position x + self class crumbSpacing) @ crumb position y.
-	self addMorphBack: crumb; updateWidth
\ No newline at end of file
+	self defaultCrumb: (self createCrumb: ALGCore title).
+	self defaultCrumb center: self center.
+	self defaultCrumb 
+		position: (self position x + self class crumbSpacing) @ self defaultCrumb position y.
```

Comment:
```suggestion
	self defaultCrumb 
		center: self center;
		position: (self position x + self class crumbSpacing) @ self defaultCrumb position y.
```

Reply:
I am not sure if this will work. @Olliwehr I think we had a reason to separate this in that manner, didn't we? What's your opinion?

Reply:
I suppose it works as the `center` statement is definively executed first and that is what we depend on right?